src/main/java/com/saintsrobotics/swerveDrive/config/OI.java

```diff
@@ -1,4 +1,4 @@
-package com.saintsrobotics.swerveDrive.input;
+package com.saintsrobotics.swerveDrive.config;
```

Comment:
As part of doing this should we also rename `swerveDrive` in the package name to something to match the repo, like `shoppingkart`?